scripts/jenkins-yoctobuild-init-script.sh

```diff
@@ -64,12 +64,13 @@ pip2 install pycrypto --upgrade
 
 # Python 3 pip
 apt_get -qy --force-yes install python3-pip
-pip3 install pyyaml --upgrade
+wget https://raw.githubusercontent.com/mendersoftware/integration/b4f9871e578fee102c6e316e19cc3a602d2a70ed/extra/requirements.txt
+pip3 install -r requirements.txt
 
```

Comment:
This file is not in use any more by us. Our Jenkins (if every resurrected) will use a `jenkins` branch and I think CFEngine team uses this file (from master) and that is the reason to keep it around.

In any case, drop the update from here because we cannot verify the side effects.